{"count":8,"days":[{"date":"2021-09-25T00:00:00.000Z","count":0,"incidents":[],"order":1},{"date":"2021-09-24T00:00:00.000Z","count":0,"incidents":[],"order":2},{"date":"2021-09-23T00:00:00.000Z","count":0,"incidents":[],"order":3},{"date":"2021-09-22T00:00:00.000Z","count":0,"incidents":[],"order":4},{"date":"2021-09-21T00:00:00.000Z","count":0,"incidents":[],"order":5},{"date":"2021-09-20T00:00:00.000Z","count":0,"incidents":[],"order":6},{"date":"2021-09-19T00:00:00.000Z","count":1,"incidents":[{"id":"0d144f54","title":"Major Outage","date":"2021-09-19T07:30:00.000Z","modified":"2021-09-23T20:35:00.000Z","scheduled":null,"severity":"major-outage","affectedsystems":["director","cluster"],"resolved":true,"content":"<p>Due to power issues, these services and the sites that run on them including all student sites and JupyterHub are unaccessable.</p>\n<!--- language code: en -->\n"}],"order":7},{"date":"2021-09-18T00:00:00.000Z","count":0,"incidents":[],"order":8}],"daysSinceLatest":6}